rustyms/src/shared/glycan.rs

```diff
@@ -25,6 +44,8 @@ impl MonoSaccharide {
             base_sugar: sugar,
             substituents: substituents.to_owned(),
             furanose: false,
+            epi: false,
```

Comment:
What does `epi` refer to?

Reply:
The `e` in `eLeg` if you know what that stands for I would be happy to use a more descriptive name, but I could not find it.

Reply:
The IUPAC trivial name is misleading. I used the more verbose nomenclature taken from these two sources:
https://pubchem.ncbi.nlm.nih.gov/compound/Legionaminic-acid
https://pubchem.ncbi.nlm.nih.gov/compound/4-Epilegionaminic-acid

`5,7-Diamino-3,5,7,9-Tetradeoxy-D-glycero-D-[**galacto**]-non-2-ulosonic Acid`
`5,7-Diamino-3,5,7,9-Tetradeoxy-D-glycero-D-[**talo**   ]-non-2-ulosonic Acid`

They refer to named sets of stereo-center arrangements in a chain of carbons that make up a monosaccharide's backbone. 

If the IUPAC trivial notation weren't so convenient, it'd be better ignored, but it's (almost) everyone's favorite shorthand.

More tomorrow.